build/package/scripts/build-typescript.sh

```diff
@@ -89,11 +97,12 @@ fi
 
 echo "Building ..."
 npm run build
-mkdir -p "${OUTPUT_DIR}/dist"
-cp -r dist "${OUTPUT_DIR}/dist"
+cp -r "${BUILD_DIR}" "${OUTPUT_DIR}/dist"
```

Comment:
I think `mkdir -p "${OUTPUT_DIR}/dist"` may be needed if the output dir does not exist?

Reply:
Actually @kuebler and I found out that with `mkdir -p docker/dist` the output will be stored inside `docker/dist/dist/...`. Only when removing the `mkdir ...` command we were able to get this to work.

Reply:
Oh haha. Then `mkdir -p docker`?

BTW: Isn't there a test that would have caught if things are not in `docker/dist`, but `docker/dist/dist`? There should be :)

Reply:
have you tried doing `cp -r dist/ "${OUTPUT_DIR}/dist"` or with `dist/*` ?

Reply:
> BTW: Isn't there a test that would have caught if things are not in docker/dist, but docker/dist/dist? There should be :)

There was none so far, but I added this as soon I realized what is happening

> have you tried doing cp -r dist/ "${OUTPUT_DIR}/dist" or with dist/* ?

`dist/` we tried without success, but `dist/*` is worth a shot

Reply:
Careful with `*`. The use case you show might work, but `cp foo/*` misses files starting with a dot.